state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -590,11 +621,27 @@ pub mod pallet {
 							details.deposit_channel.channel_id,
 							DepositChannel { state, ..details.deposit_channel },
 						);
+						used_weight = used_weight.saturating_add(
+							frame_support::weights::constants::RocksDbWeight::get()
+								.reads_writes(1, 0),
+						);
 					}
+					// Cleanup the deposit data
```

Comment:
Does having items here indicate that a deposit was pre-witnessed, but never fully witnessed? I wonder if we want to log this somehow.

Reply:
Yes it does. I think it might only matter if it was boosted.